docs/.vuepress/config.ts

```diff
@@ -19,6 +19,7 @@ export default defineUserConfig({
   alias: {
     // Alias src directory for @/components/{KongponentName} imports
     '@': path.resolve(__dirname, '../../src'),
+    mocks: path.resolve(__dirname, '../../mocks'),
```

Comment:
Since this is an alias, let's utilize `@` but that will require adjusting the other alias as well (you'll need to make sure this doesn't break anything)
```suggestion
    '@/': path.resolve(__dirname, '../../src/'),
    '@mocks/': path.resolve(__dirname, '../../mocks/'),
```